cmd/cluster/from_infra_id.go

```diff
@@ -79,10 +81,29 @@ func (ops *fromInfraIdOptions) run(cmd *cobra.Command, args []string) error {
 			return renderOutput(cluster, ops.globalOpts.Output)
 		}
 	}
-	_, err = fmt.Fprintf(os.Stderr, "No clusters found matching %s\n", infraId)
+	_, _ = fmt.Fprintf(os.Stderr, "No clusters found matching %s\n", infraId)
```

Comment:
Would you be opposed to wrapping this in a check for the `verbose` flag?  "Checking for" messages are useful, but implied by the fact that the next function you call will return that context.  Leaning toward UNIX "no news is good news".  

I'll agree that it's less likely with this particular scenario that folks might be piping to another command to do something with the data, but if they did, having informational info behind the verbose flag helps for parsing output.

Reply:
Hm, let me think about that. I like what you're saying. Right now, this prints to stderr so pipelines wouldn't ingest it. Maybe that's sufficient? 🤔

Reply:
Updated

Reply:
We chatted offline and my latest push should have sufficiently informative output given the constraint that the display name we're searching on is not guaranteed to be unique.